src/pages/LoginPage.js

```diff
@@ -31,37 +41,64 @@ class LoginPage extends React.Component {
         this.props.createToken({variables: mutationVariables}).then((response) => {
             let token = response.data.createToken.token;
             this.props.currentUserActions.setCurrentUserToken(token);
-            if(this.props.currentUserToken !== "N/A"){
+            if (this.props.currentUserToken !== "N/A") {
+                this.setState({loginFailed: false});
                 // DISPLAY SUCCESS / REDIRECT
+                this.setState({loginSuccess: true});
             }
             // console.log(token);
         }).catch((error) => {
             console.log("Caught an error");
```

Comment:
This error message should be more  descriptive. In the console logs, it's not guaranteed to appear next to the error printout given below.